src/vm/tests/test-update.js

```diff
@@ -1480,33 +1480,36 @@ test('add fs /var/tmp/global', function (t) {
             t.ok(false, 'error updating VM: ' + update_err.message);
             t.end();
         } else {
-            VM.load(vm_uuid, function (err, obj) {
-                var field;
+            setTimeout(function () {
+                t.ok(true, 'waited 1s for vminfod');
```

Comment:
please either update the value here, or maybe remove the message?

Reply:
I'll remove it, it adds no value.